src/Cake.Core/Cake.Core.csproj

```diff
@@ -131,11 +131,21 @@
     <Compile Include="IO\IRegistry.cs" />
     <Compile Include="IO\IRegistryKey.cs" />
     <Compile Include="IO\Machine.cs" />
+    <Compile Include="IO\NuGet\AssemblyFilePathFrameworkNameParser.cs" />
```

Comment:
I was thinking more of grouping code functionality. That's how we done it
so far.
On Nov 9, 2015 18:01, "James Nail" <notifications@github.com> wrote:

> In src/Cake.Core/Cake.Core.csproj
> <https://github.com/cake-build/cake/pull/435#discussion_r44299783>:
>
> > @@ -131,11 +131,21 @@
> >      <Compile Include="IO\IRegistry.cs" />
> >      <Compile Include="IO\IRegistryKey.cs" />
> >      <Compile Include="IO\Machine.cs" />
> > +    <Compile Include="IO\NuGet\AssemblyFilePathFrameworkNameParser.cs" />
>
> Depends on whether you care about using namespaces to denote "components"
> (where you'd typically allow types to use other types that are higher in
> the namespace hierarchy, and prohibit using those that are lower) versus
> simply grouping code functionally. I'm open to whatever you prefer.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cake-build/cake/pull/435/files#r44299783>.
>